operator/v1/types_network.go

```diff
@@ -321,6 +321,10 @@ type OVNKubernetesConfig struct {
 	// not using OVN.
 	// +optional
 	HybridOverlayConfig *HybridOverlayConfig `json:"hybridOverlayConfig,omitempty"`
+	// EnableIPsec enables IPsec encryption on all pod-pod traffic within the cluster.
+	// Default is false.
+	// +optional
+	EnableIPsec bool `json:"enableIPsec"`
```

Comment:
sorry for the late comment. Actually, to make this discoverable, you have to add a default `kubebuilder:default`, compare https://book.kubebuilder.io/reference/markers/crd-validation.html.

Reply:
and a last nit: `EnableIPsec` in the godoc must be lower-case

Reply:
do we need `omitempty` on this?

Reply:
i guess the recommendation from @sttts was to drop the omitempty in favor of making the flag more discoverable. But if we move that into an entirely new struct `IPsecConfig` and mark that config variable `omitempty`, is that OK? i.e. absence of the config in the json struct a reasonable way to default the ipsec config to false?